### DISCUSSION TITLE
Site overview, groups - ID vs. recordnumber

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#360] [General]
Had the issue that wrong groups are presented in the overview. Even when I tried to filter the view a wrong result came up.

Found out, that I had deleted the first group. Means a group with ID=1 didn't exist any more. Created again a group named testsites - it became the next free id (here 6) and still the same. Than I changed it in the database to ID=1. After this all was working as expected.

Have the impression for presentation of the groups in the view not the ID is used, instead of that the record number. Which is not identical after a deletion.

Think not a big issue,

**Comment by nikosdion** [comment]
The problem is different. The filters work correctly. It's editing a site that's wrong. This is where the assignment was wrong when there are gaps in sequential numbering of groups.

I have fixed that for version 1.0.6 which will be released later this week.
